doc/schemas/commando-rune.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
```

Comment:
Can be `false`?  No extra stuff is ever added it seems.

Reply:
Indeed, wonder where I cut & paste that from?